src/bootstrap/src/core/build_steps/llvm.rs

```diff
@@ -827,6 +818,16 @@ fn configure_llvm(builder: &Builder<'_>, target: TargetSelection, cfg: &mut cmak
         }
     }
 
+    // Libraries for ELF compression.
```

Comment:
Maybe something like the following, for people unfamiliar with all this?

```suggestion
    // Libraries for ELF debug sections compression.
```

Reply:
I went with "ELF section compression" since it's theoretically at least not limited to debug stuff.